dist/schema.json

```diff
@@ -206,6 +206,18 @@
                 "type": "string"
               }
             }
+          },
+          {
+            "name": "id",
```

Comment:
`id`on `GET /api/v1/accounts` as actually listed under `Query parameters`. It should not match as a header.